.github/workflows/release-swift-toolchain-binary-sizes.yml

```diff
@@ -0,0 +1,155 @@
+# Uploads a toolchain release's binary size metrics to DataDog.
+#
+# To target a specific Swift Toolchain Release, run:
+#
+#     $TOOLCHAIN_VERSION="..."
+#     gh workflow run "Release - Swift Toolchain Binary Sizes" `
+#       -f toolchain_version=${TOOLCHAIN_VERSION} `
+#       -f environment_label=${USER} `
+#       -R github.com/thebrowsercompany/swift-build `
+#
+name: Release - Swift Toolchain Binary Sizes
+
+# TODO(thebrowsercompany/swift-build/issues/129): Support arm64 releases.
+# TODO(kendal): Use on.releases.[created, edited] when the table schema is stablized.
+
+on:
+  workflow_dispatch:
+    inputs:
+      toolchain_version:
+        description: 'Use this swift toolchain release version'
+        required: false
+        type: string
+        default: '' # See env.SWIFT_TOOLCHAIN_VERSION
+      environment_label:
+        description: 'Tag the uploaded data with this value. This helps with filtering'
+        required: false
+        type: string
+        default: 'debug'
```

Comment:
Why is it debug by default?

Reply:
I'm not sure I understand why delete it in a subsequent PR rather than this PR.

Reply:
I still use it. I'll also use it as a tag when we start uploading metrics to DataDog.

Reply:
If it is arbitrary, why not make it a guid?

Reply:
Hmm, I suppose we could but I don't think think it's worth spending the time change the table schema again :/ this field is only being used in the interim while we get this pipeline up and running.

Reply:
Why is the value encoded into the schema?

Reply:
sync from offline: It's just a tag for human filtering - the default being 'debug' is confusing.

I removed the default - this field is nullable in BQ and should only be set when testing/debugging.